model/config.go

```diff
@@ -296,6 +296,7 @@ type ServiceSettings struct {
 	EnablePostUsernameOverride                        *bool    `access:"integrations"`
 	EnablePostIconOverride                            *bool    `access:"integrations"`
 	EnableLinkPreviews                                *bool    `access:"site"`
+	RestrictLinkPreviews                              *string  `access:"site"`
```

Comment:
@wiersgallak shall we add this new setting to the telemetries?

Reply:
@isacikgoz Yes please